Services/data/devicemodels/scripts/TempIncrease-method.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+/*global log*/
+/*global updateState*/
```

Comment:
To match the latest js file format, should also include: `/*global updateProperty*/`